src/index.ts

```diff
@@ -193,12 +193,16 @@ export async function main(
   }
 
   let coverageFilePaths: string[] = []
-  if (args.file) {
+  if (args.file !== undefined) {
```

Comment:
This condition previously skipped empty paths if:
- only one occurrence of the `-f` flag was used; and
- the value passed to it was an empty string.